docs/resource/k8s.md

```diff
@@ -531,3 +531,4 @@ A ResourceQuota provides constraints that limit aggregate resource consumption p
 The following guidelines are specific to Kubernetes:
 
 - [Service in Kubernetes](../non-normative/k8s-attributes.md#service-attributes)
+- [Additional naming guidance for Kubernetes](../non-normative/naming-known-exceptions.md#kubernetes-naming-exceptions)
```

Comment:
All: just a reminder that links should _not_ point to page level 1 headings, because those don't exist in the published website. To see how this breaks the website build, visit https://github.com/open-telemetry/opentelemetry.io/actions/runs/15373995491/job/43256707459?pr=6948.

I'll submit a fix shortly.